### PR TITLE
Feature/17 new warehouse WIP

### PR DESCRIPTION
**Pull request by trumerfe**

### CODE REVIEW

**Review by jwuzzle — APPROVED** [review]

**Review by sanchezfernandes — APPROVED** [review]
Rockstar code man

**Review by Cob007 — APPROVED** [review]
Excellent 💪🏾

File: src/components/FormFields/FormFields.scss [hunk 1]
```diff
@@ -0,0 +1,32 @@
+@use '../../partials/globals' as *;
+@use '../../partials/mixins' as *;
+@use '../../partials/variables' as *;
+
+.form {
+  margin-bottom: 16px;
+
+  &__field-label {
+    @include labels-links-buttons;
+    // margin-bottom: 4px;
```
Comment (Edthev) [inline src/components/FormFields/FormFields.scss:10]: is this comment needed?

File: src/pages/NewWarehouse/NewWarehouse.jsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import "./NewWarehouse.scss";
+import ArrowBack from "../../assets/Icons/arrow_back-24px.svg";
+import FormFields from "../../components/FormFields/FormFields";
+import { Link } from "react-router-dom";
+
+const NewWarehouse = () => {
+
+  const onSubmit = (event) => {
+    event.preventDefault()
```
Comment (Edthev) [inline src/pages/NewWarehouse/NewWarehouse.jsx:9]: does this reset the input?
`event.target.reset`

Comment (Cob007) [reply src/pages/NewWarehouse/NewWarehouse.jsx:9]: we can use useState approach on the form and put the func inside onClick and the onChange on the input dom elements @trumerfe @Edthev

**Pull request merged**